lrrbot/commands/quote.py

```diff
@@ -39,16 +39,17 @@ def format_quote(tag, qid, quote, name, date, context):
 		quote_msg += " [{date!s}]".format(date=date)
 	return quote_msg
 
-@bot.command("quote(?: (?:(\d+)|(.+)))?")
+@bot.command("quote(?: (?:(game|show) (.+)|(?:(\d+)|(.+))))?")
 @lrrbot.decorators.sub_only
 @lrrbot.decorators.throttle(60, count=2)
-def quote(lrrbot, conn, event, respond_to, qid, attrib):
+def quote(lrrbot, conn, event, respond_to, meta_param, meta_value, qid, attrib):
 	"""
 	Command: !quote
 	Command: !quote ATTRIB
+	Command: !quote game|show VALUE
```

Comment:
This should be two entries `!quote game GAME` and `!game show SHOW` to match how other commands are described.